### PR TITLE
Updated: Slogan & Network Status

### PR DESCRIPTION
**Pull request by GoliathLabs**
The slogan has been changed to the suggestion from @krombel.
The formatting of the network status area on our main page has been changed.

### CODE REVIEW

File: _includes/network-status.md [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
 <br>
 <br><u><b>Aktuell:</b></u>
+<br>Aktuell sind uns keine Störungen bekannt
 <br>
 <br><u><b>History:</b></u>
-<br><b>01.10.20 10:20-13:46</b> Ausfall von DoT (dot.ffmuc.net)
-<br><b>04.05.20 17:20-17:50</b> IPv4 DHCP gestört für ca. 30 Minuten
-<br><b>03.06 - 06.06.2019</b> Mailserver offline. u.A. Tickethandling, Registrierungen im Chat sind nicht möglich
-<br><b>27.03.2019 17:00-20:00</b> Das gesamte Netz ist aufgrund eines geplanten Serverumzugs offline
+<br><b>01.10.2020 10:20 - 13:46</b> Ausfall vom DoT (dot.ffmuc.net)
```
Comment (awlx) [inline _includes/network-status.md:8]: I say `von` is correct but as it's english in a german sentence whatevs.

**Review by awlx — APPROVED** [review]

File: _includes/network-status.md (hunk 1 above)
Comment (GoliathLabs) [reply _includes/network-status.md:8]: Alright :D

File: _includes/network-status.md [hunk 2]
```diff
@@ -2,13 +2,14 @@
...
 <br>
 <br><u><b>Aktuell:</b></u>
+<br>Aktuell sind uns keine Störungen bekannt
 <br>
 <br><u><b>History:</b></u>
-<br><b>01.10.20 10:20-13:46</b> Ausfall von DoT (dot.ffmuc.net)
-<br><b>04.05.20 17:20-17:50</b> IPv4 DHCP gestört für ca. 30 Minuten
-<br><b>03.06 - 06.06.2019</b> Mailserver offline. u.A. Tickethandling, Registrierungen im Chat sind nicht möglich
-<br><b>27.03.2019 17:00-20:00</b> Das gesamte Netz ist aufgrund eines geplanten Serverumzugs offline
+<br><b>01.10.2020 10:20 - 13:46</b> Ausfall des DoT (dot.ffmuc.net)
```
Comment (awlx) [inline _includes/network-status.md:8]:
```suggestion
<br><b>01.10.2020 10:20 - 13:46</b> Ausfall des DoT Servers (dot.ffmuc.net)
```

Comment (awlx) [reply _includes/network-status.md:8]: Like this? Dunno ...

Comment (GoliathLabs) [reply _includes/network-status.md]: Looks good to me!

**Pull request merged**